### PR TITLE
Swift at root URL checks

### PR DESCRIPTION
**Pull request by TejasC88**
Operations:
1. Set the rgw_swift_url_prefix
2. Check the swift at "/" urls
3. With swift at root set , check S3 access is not allowed in the same namespace.

Pass log : http://magna002.ceph.redhat.com/cephci-jenkins/tchandra/swift_at_root.txt

### CODE REVIEW

File: rgw/v2/tests/s3_swift/test_swift_basic_ops.py [hunk 1]
```diff
@@ -801,6 +802,62 @@ def test_exec(config, ssh_con):
...
+            if srv_restarted is False:
+                raise TestExecError("RGW service restart failed")
+            else:
+                log.info("RGW service restarted")
+            log.info("Check the swift url works at root")
+            ip_and_port = rgw.authurl.split("/")[2]
+            proto = "https" if config.ssl else "http"
+            url = f"{proto}://{ip_and_port}/"
+
+            log.info("CHeck swift /info")
```
Comment (ckulal) [inline rgw/v2/tests/s3_swift/test_swift_basic_ops.py:822]: Minor: typo

**Review by ckulal — COMMENTED** [review]
other than minor comment,
remaining changes looks good

File: rgw/v2/tests/s3_swift/test_swift_basic_ops.py [hunk 2]
```diff
@@ -51,6 +51,7 @@
 import traceback
 
 import names
+import requests
 import v2.lib.manage_data as manage_data
 import v2.lib.resource_op as swiftlib
 import v2.utils.utils as utils
```
Comment (anrao19) [inline rgw/v2/tests/s3_swift/test_swift_basic_ops.py:57]: Please update the config file : configs/test_swift_at_root.yaml

Comment (TejasC88) [reply rgw/v2/tests/s3_swift/test_swift_basic_ops.py:58]: Done , pls review

File: rgw/v2/tests/s3_swift/test_swift_basic_ops.py (hunk 1 above)
Comment (TejasC88) [reply rgw/v2/tests/s3_swift/test_swift_basic_ops.py]: Done , pls review

**Review by ckulal — COMMENTED** [review]
LGTM

**Review by hmaheswa — APPROVED** [review]

**Review by anrao19 — APPROVED** [review]
LGTM

**Review by viduship — APPROVED** [review]

**Review by ckulal — APPROVED** [review]

**Pull request merged**